docs/configuring.md

```diff
@@ -164,6 +164,8 @@ Some Linux kernels implement Transparent HugePage Support (THP), which automates
 - The `-Xlp:objectheap` option requests that the Java object heap is allocated by using large pages.
 - The `-Xlp:codecache` option requests that the JIT code cache is allocated by using large pages.
 
+    If the configured large page size is greater than the size of the total JIT code cache, the next available lower page size on the platform is used for the code cache allocation.
```

Comment:
So, how do we enable large pages on the system? As explained in the https://eclipse-openj9.github.io/openj9-docs-staging/1242/configuring/#configuring-large-page-memory-allocation section -> for different systems?

There is also another line in that topic

>> You must also enable large pages on your local system. This process differs according to the operating system.

- In addition to the local system, where else are we supposed to enable the large pages? 
- What is meant by "This process differs according to the operating system."? As explained in the https://eclipse-openj9.github.io/openj9-docs-staging/1242/configuring/#configuring-large-page-memory-allocation section -> for different systems? If so, then is this for enabling large pages on the local system? 
- What is the process to enable large pages on systems other than the local system?

Thanks!

Reply:
To further elaborate, using large pages in the JVM requires two things
1) Enable large page support in the OS for the machine to be used. This can be done as explained in the articles above
2) Instruct the JVM to take advantage of large pages. This can be done with JVM specific options (-Xlp), if the JVM doesn't already do it by default.

I am not talking here about THP (transparent huge pages) on Linux which tries to use large pages (if allowed) transparently without user/application intervention.